pkg/projects/Microsoft.NET.CoreRuntime/AppxManifest.xml.template

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+   xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+   xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+   IgnorableNamespaces="uap"> 
+<Identity Name="[AppxIdentityName]" 
+  ProcessorArchitecture="[AppxBuildArch]" 
+  Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" 
+  Version="[AppxVersion]" />
+<mp:PhoneIdentity PhoneProductId="7D3B4B8B-CA47-4518-823A-A7793F27DA93" 
+  PhonePublisherId="6B151E87-785E-4C4A-AA41-823B05AA9A66" />
+<Properties>
+  <Framework>true</Framework>
+  <DisplayName>Microsoft .Net Core Runtime Package 2.0</DisplayName>
```

Comment:
We should set the version number using the information coming from branchinfo.txt (similar to how AppXVerison and AppXIdentityName are being set earlier via parameters.